hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -239,6 +239,15 @@ private boolean isBlockCorrupt(int blocksize) throws IOException {
       return true;
     }
 
+    // check if the blocksize mentioned in the footer is the same as the header; by seeking back the length of a long
+    // the backward seek does not incur additional IO as {@link org.apache.hadoop.hdfs.DFSInputStream#seek()}
+    // only moves the index. actual IO happens on the next read operation
+    inputStream.seek(inputStream.getPos() - Long.BYTES);
```

Comment:
This is assuming that the block length will always be the last thing written in the footer, can we add comments in the footer writer part to denote this

Reply:
added comment to the write